samples/excel/tables/apply-values-filter.yaml

```diff
@@ -0,0 +1,156 @@
+id: ''
+gist: ''
+source: EXCEL
+author: Microsoft
+name: Apply a custom filter
+description: Apply a custom filter to a table
+script:
+  content: |
+    $('#create-table').click(createTable);
+    $('#filter-data').click(filterData);
+
+    /** Create a new table with some sample data */
+    function createTable() {
+        Excel.run(async (context) => {
+            let sheet: Excel.Worksheet;
+            try {
+                // If sheet already exists, queue a command to delete it
+                //context.workbook.worksheets.getItem("Playground").delete();
+
+                // Then create a proxy object for a new worksheet
+                sheet = context.workbook.worksheets.add("Playground");
+
+                // Call the helper to create the table
+                await queueCreateTable(sheet);
+
+                //Queue a command to activate the sheet
+                sheet.activate();
+
+                // Run all of the queued up commands
+                await context.sync();
+            }
+            catch (error) {
+                // Re-throw the original error, for appropriate error-handling
+                // (in this snippet, simply showing a notification)
+                throw error;
+            }
+        })
+        .catch(handleError);
+    }
+
+
+    /** Filter the data in the table*/
+    function filterData() {
+        Excel.run(async (context) => {
+            let sheet: Excel.Worksheet;
+            try {
+                // Queue a command to get the sheet
+                sheet = context.workbook.worksheets.getItem("Playground");
+
+                // Call the helper to sort the table
+                await queueFilterData(sheet);
```

Comment:
I would prefer that the code is brought inline and not placed in a helper function